react-hot-button-component/src/hot-button.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+class HotButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      numberOfClicks: 0
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    this.setState({ numberOfClicks: this.state.numberOfClicks + 1 });
+  }
+
+  render() {
+    const numberOfClicks = this.state.numberOfClicks;
+    let className;
+    if (numberOfClicks >= 18) {
```

Comment:
While this works well, I would move the logic for the class selection to a separate method to clean up the render method. You can just call the method then and save the proper className in your className variable.